src/include/execution/compiler/operator/hash_join_translator.h

```diff
@@ -37,12 +37,20 @@ class HashJoinTranslator : public OperatorTranslator {
                      Pipeline *pipeline);
 
   /**
-   * Declare the build-row struct used to materialized tuples from the build side of the join.
-   * @param decls The top-level declarations for the query. The build-row struct will be
-   *                        registered here after it's been constructed.
+   * Declare the build-row struct and probe-row struct used to materialized tuples from the build side
```

Comment:
Fix these comments to reflect under what conditions structs are actually generated.